blockchain_exploration/types.py

```diff
@@ -4,24 +4,13 @@ class Network:
     BITCOIN_CASH = "bitcoin-cash"  # https://bitcoincash.org/
     ETHEREUM = "ethereum"  # http://ethereum.org/
     MATIC = "matic"  # https://polygon.technology/
-    POLYGON = "matic"  # Polygon was formerly named Matic; and this value is spread around our databases
```

Comment:
duplicate entries for MATIC and POLYGON.  using MATIC since is the most widely used one.